### PR TITLE
unpack/repack orderedDict

### PR DESCRIPTION
**Pull request by justinwp**
- [x] Tests added / passed
- [ ] Passes `flake8 dask`

### CODE REVIEW

**Comment by mrocklin** [comment]
cc @jcrist

File: dask/base.py [hunk 1]
```diff
@@ -263,8 +263,8 @@ def _unpack(expr):
             typ = list if isinstance(expr, Iterator) else type(expr)
             if typ in (list, tuple, set):
                 tsk = (typ, [_unpack(i) for i in expr])
-            elif typ is dict:
-                tsk = (dict, [[_unpack(k), _unpack(v)]
+            elif typ in [dict, OrderedDict]:
```
Comment (jcrist) [inline dask/base.py]: Slight preference to use a tuple here instead of a list (style nit). Otherwise this LGTM.

**Comment by jcrist** [comment]
Draft PRs are new to me, looks like they were added a month ago (reference for others): https://github.blog/2019-02-14-introducing-draft-pull-requests/

**Comment by justinwp** [comment]
>Draft PRs are new to me, looks like they were added a month ago (reference for others): https://github.blog/2019-02-14-introducing-draft-pull-requests/

My typical use has been to leave in draft until I get CI passing unless it is sufficiently complex to warrant review earlier.

**Comment by jrbourbeau** [comment]
FWIW just as a side note, draft PRs don't currently trigger Travis CI builds (although it appears there's some internal discussion about if they want to change this https://travis-ci.community/t/draft-pull-requests-not-being-built/2434)

**Comment by jcrist** [comment]
LGTM. Thanks @justinwp.

**Comment by jrbourbeau** [comment]
Thanks @justinwp. I noticed this was your first code contribution to Dask, welcome!

**Pull request merged**